db/migrate/20150915155404_change_service_request_quote_status_to_get_cost_estimate.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeServiceRequestQuoteStatusToGetCostEstimate < ActiveRecord::Migration
+  def up
+    ServiceRequest.where(status: 'get_a_quote').update_all status: 'get_a_cost_estimate'
```

Comment:
I think you need to do this on SubServiceRequest, AvailableStatus, and PastStatus.  They all have a status column